lib/generators/rails/mailbox_generator.rb

```diff
@@ -19,8 +17,6 @@ def create_mailbox_file
             template "application_mailbox.rb", application_mailbox_file_name
           end
         end
-
-        template "mailbox_test.rb", File.join('test/mailboxes', class_path, "#{file_name}_mailbox_test.rb")
```

Comment:
This will mean that we won’t generate a test stub when you’re actually generating a new mailbox though, right?

Reply:
Maybe the root application_controller.rb can’t actually use the generator itself.

Reply:
We are, I moved it [here](https://github.com/rails/actionmailbox/pull/9/files#diff-72281b3893d81eac3535fb1b4ebe11b8R3)... Problem was that this [line](https://github.com/rails/actionmailbox/pull/9/files#diff-1c1c483b957dca20f2c9c7cafd416168R22), was throwing error in console, since no test framework was hooked

<img width="1242" alt="screenshot 2018-12-15 at 16 50 45" src="https://user-images.githubusercontent.com/7427365/50044806-9c0b0c00-0089-11e9-878f-8ceb8f4e832f.png">

Reply:
...with this approach we can then kill generating `test/mailboxes/application_mailbox_test.rb` on `bin/rails action_mailbox:install`
https://github.com/rails/actionmailbox/pull/9/files#diff-7b2c92fa1c2cf55acbae4838c02d2944R11

Reply:
this was the error I was getting using the old way 
<img width="561" alt="screenshot 2018-12-15 at 16 59 26" src="https://user-images.githubusercontent.com/7427365/50044920-e80a8080-008a-11e9-8a6e-624b38a2dd83.png">

Reply:
👌